### PR TITLE
Add text on observations

### PR DESCRIPTION
**Pull request by jondequinor**
**Issue**
#767 

**Approach**
- Allows for a `str` -> `str` dict on `Observations`

### CODE REVIEW

File: ert_shared/storage/http_server.py [hunk 1]
```diff
@@ -110,6 +113,34 @@ def data(self, data_id):
...
+            rdb_url=self._rdb_url, blob_url=self._blob_url
+        ) as api:
+            return api.observation(name)
+
+    def get_observation_tags(self, name):
+        with StorageApi(
+            rdb_url=self._rdb_url, blob_url=self._blob_url
+        ) as api:
+            try:
+                return {"tags": api.observation_tags(name)}
```
Comment (ManInFez) [inline ert_shared/storage/http_server.py:127]: I think it is a good idea we move all generation of return schema to the storage_api, unless it directly relates to http stuff (see resolve_urls)

File: ert_shared/storage/http_server.py [hunk 2]
```diff
@@ -110,6 +113,34 @@ def data(self, data_id):
...
+            except KeyError:
+                return {}
```
Comment (ManInFez) [inline ert_shared/storage/http_server.py:129]: if the name is not found should we not return a 404 resonse?

File: ert_shared/storage/model.py [hunk 3]
```diff
@@ -237,12 +245,49 @@ def __repr__(self):
...
+        Observation,
+        backref=backref(
+            "observations_attributes",
+            collection_class=attribute_mapped_collection("attribute"),
+            cascade="all, delete-orphan",
+        ),
+    )
+
+
+class Value(Entities):
```
Comment (ManInFez) [inline ert_shared/storage/model.py:278]: If this is only used by observation_attributes, is it maybe a bit too generically named?

File: ert_shared/storage/rdb_api.py [hunk 4]
```diff
@@ -248,6 +249,17 @@ def add_observation(
 
         return observation
 
+    def add_observation_attribute(self, name, attribute, value):
+        msg = "Adding {} with value {} to observation with name {}"
+        logging.info(msg.format(attribute, value, name))
```
Comment (ManInFez) [inline ert_shared/storage/rdb_api.py:254]: 👍 for logging

**Comment by jondequinor** [comment]
Jenkins test this please

**Comment by jondequinor** [comment]
Test this please!

**Comment by jondequinor** [comment]
Test this please!

**Comment by jondequinor** [comment]
Test this please!

**Comment by jondequinor** [comment]
Test this please!

**Pull request merged**